public/assets/main.css

```diff
@@ -74,34 +108,66 @@ a.game__nav:focus {
   }
 }
 
+#profile-settings {
```

Comment:
my only other comment is this css, i don't know what it does because i don't know CSS very well.

is it necessary to put this here vs inline with functions that wrap these divs?

Reply:
You’d have to put style-declarations in at least 8 places in the code for this to work. To me that’s way too cumbersome. As is I can view html and css side by side without scrolling which makes for easier layouting. Once that’s done you can of course move everything back to the code. But it most definitely doesn’t help future **front end** devs who’d like to or have to change CSS in the future …